### ISSUE TITLE
GPU?

### ISSUE THREAD

**Issue by Driadasasa** [UnamSanctam/SilentCryptoMiner#1781]
https://dropmefiles.com/mOoyy
Hello, Unam. How does this work? Does it need to be enabled or not?

**Comment by Tamshi27** [comment]
Well, if you want to mine on the gpu then you need to enable it, but keep in mind if you mine a coin like xmr dont enable it bc its a only cpu coin

**Comment by Driadasasa** [comment]
> Well, if you want to mine on the gpu then you need to enable it, but keep in mind if you mine a coin like xmr dont enable it bc its a only cpu coin

I don't get it, I'm mining XMR and I thought there's some neurotechnology here. The miner determines what card and processor and therefore distributes the load for better mining xD

**Comment by UnamSanctam** [comment]
Monero is designed (the cryptocurrency itself) to be unprofitable for GPU mining in order for CPU mining to be able to compete. So for GPU mining it's best to mine a cryptocurrency that's designed for GPU mining, such as ETC, RVN or something else. You can compare profitability between GPU coins on this page https://whattomine.com/.

**Comment by FF5252** [comment]
Which coin is good for low-end GPUS

**Comment by FF5252** [comment]
I dont find nowhere the DAG size of coins only on MinerStat

**Comment by opget** [comment]
how to set up ETH classic mining on 2miners.com in real life I get a red "Error" notification in the unampanel. I am using these settings, please advise me if I am doing something wrong. Here is an excerpt from the config to connect the miner to the pool

  ```
  { "url": "stratums://`0xHEREWALLETADRESS`:x@asia-etc.2miners.com:1010",
    "algo": "etchash",
```

> Monero (сама криптовалюта) спроектирована так, чтобы быть невыгодной для майнинга на GPU, чтобы майнинг на CPU мог конкурировать. Поэтому для майнинга с помощью графического процессора лучше всего использовать криптовалюту, предназначенную для майнинга с помощью графического процессора, например ETC, RVN или что-то еще. Вы можете сравнить прибыльность монет GPU на этой странице https://whattomine.com/ .

**Comment by UnamSanctam** [comment]
>Which coin is good for low-end GPUS
>I dont find nowhere the DAG size of coins only on MinerStat

Yes that minerstat page is probably the only one that shows DAG sizes haha, to know the DAG sizes of other coins you have to test it yourself. But if you want to mine a coin with a low DAG size then you can mine Zano or Meowcoin, both only need around 1GB of VRAM.

>how to set up ETH classic mining on 2miners.com in real life I get a red "Error" notification in the unampanel. I am using these settings, please advise me if I am doing something wrong. Here is an excerpt from the config to connect the miner to the pool

Since you have SSL enabled (`stratums`) then you should use their SSL port `11010` instead of `1010`. Their SSL port section is a bit hidden, you can find it on the bottom of the page where they show the pool addresses and ports.

**Comment by opget** [comment]
> > Какая монета подходит для младших графических процессоров. 
> > Я нигде не нашел монет размера DAG, только на MinerStat.
> 
> Да, эта страница minerstat, вероятно, единственная, на которой показаны размеры DAG, ха-ха. Чтобы узнать размеры DAG других монет, вам нужно проверить это самостоятельно. Но если вы хотите добыть монету с небольшим размером DAG, вы можете майнить Zano или Meowcoin, для обоих требуется всего около 1 ГБ видеопамяти.
> 
> > как настроить классический майнинг ETH на 2miners.com в реальной жизни Я получаю красное уведомление «Ошибка» в unampanel. Я использую эти настройки, пожалуйста, сообщите мне, если я делаю что-то не так. Вот выдержка из конфига для подключения майнера к пулу
> 
> Поскольку у вас включен SSL ( `stratums`), вам следует использовать их порт SSL `11010`вместо `1010`. Их раздел портов SSL немного скрыт, вы можете найти его внизу страницы, где показаны адреса пулов и порты.

Thanks Unam. Fixed the port in the remote configuration as soon as the add-on is applied I will write if it fixed the problem. Another question is how often does the miner access the remote configuration?

**Comment by UnamSanctam** [comment]
>Thanks Unam. Fixed the port in the remote configuration as soon as the add-on is applied I will write if it fixed the problem. Another question is how often does the miner access the remote configuration?

It contacts it every 100 minutes to avoid getting banned.

**Comment by opget** [comment]
> > Thanks Unam. Fixed the port in the remote configuration as soon as the add-on is applied I will write if it fixed the problem. Another question is how often does the miner access the remote configuration?
> 
> It contacts it every 100 minutes to avoid getting banned.

![image](https://github.com/UnamSanctam/SilentCryptoMiner/assets/141766195/bd783ab9-4a04-49ea-8db3-19635576a9eb)
Unam, the miner got new settings and now I see this status "Stopped", what could be the problem?

**Comment by UnamSanctam** [comment]
>Unam, the miner got new settings and now I see this status "Stopped", what could be the problem?

Stopped is quite uncommon, usually it means that the pool stopped responding or something like that. If it's only temporary then it could be that it reported to the web panel while it was remaking the connection (it does so occasionally to make sure nothing stops working).

**Comment by UnamSanctam** [comment]
>GPU miner constantly shows a stopped status

Have you tried another pool?

>Unam, can you tell me why there can be such drawdowns on KH\s?

Since mining is luck/probability based then finding a share takes a "random" amount of time, which means that if you count hashrate as the amount of shares submitted over some interval (like xmrig-proxy and hashvault does) then the hashrate should go up and down. The real hashrate can then be seen by using the average over a longer period of time. You can also decrease the share difficulty in order to get shares more frequently which will make it look "smoother".

**Comment by UnamSanctam** [comment]
>I haven't tried to change the pool on GPU, can you tell me what pool to change to besides 2miners?

You can try nanopool quickly and see if it changes, it can also say "Stopped" if the GPU stopped working or was disabled in some way. Does it say that for multiple miners?

>Thanks for the detailed response.
So such drawdowns are considered normal?
Or do I need to change some settings or pool?

Yes it's normal, how big the ups and downs are depends on the pools share difficulty and how it calculates the hashrate on the pool side. In the miner itself the hashrate is almost always at a constant stable number.

**Comment by UnamSanctam** [comment]
>You can try nanopool quickly and see if it changes, it can also say "Stopped" if the GPU stopped working or was disabled in some way. Does it say that for multiple miners?

Only two of them or are there more? Some ISPs or whole countries sometimes block some pool servers. The "Error" status is almost always because it's unable to connect to the pool.
